### DISCUSSION TITLE
Sharing small image filter project

### DISCUSSION THREAD

**Discussion by amjackson11** [kovidgoyal/kitty#3606] [Show and tell]
Hi, I'm a computer science student trying to build a bit of a portfolio and get experience. Right now I'm working on a command line image editor tool that uses kitty's graphics protocol and wanted to share. I realize that Python would've made more sense to use but Go is the language I'm working on trying to master right now.

Since it's my first time putting a mini tool out there I anticipate many many bugs and would appreciate feedback (with the graphics implementation & in general). Thanks!

https://github.com/amjack100/imgkit

**Comment by kovidgoyal** [comment]
Sadly I dont read Go so its hard for me to comment on your code, but cool idea and do let me know if you have questions/encounter issues. The biggest thing is to make sure you are using the file or shared memory methods when possible for best performance.
